src/checkactivitytest.rb

```diff
@@ -0,0 +1,47 @@
+require_relative "checkActivities"
+require "test/unit"
+
+class TestCheckActivity < Test::Unit::TestCase
+
+    @@outputFile = "testoutput.txt"
+    @@expecOutputFile = "regressiontestoutput.txt"
+
+    def change_stdout(file)
+        if $stdout == STDOUT then $stdout = file
```

Comment:
I'm not sure to understand why you just don't set $stdout to file. And if you want to revert to the real stdout, call the function with STDOUT?

Reply:
I'm sorry, i actually don't remember why i did that. :P 
I've changed it now.